opdatasigverify.md

```diff
@@ -0,0 +1,122 @@
+# Enable oracle-based data import via OP_DATASIGVERIFY
+
+Version 0.2, 2018-02-22
```

Comment:
We probably need a copyright and original author section at this point since this was originally in another repository.

Reply:
Wonder if we could also modify this commit so that authorship is attributed to @gandrewstone.